docs/develop/sdk/awakeables.mdx

```diff
@@ -34,15 +34,7 @@ You can use awakeables in the SDK by doing the following:
 <TabItem value="ts" label="TypeScript">
 
 ```typescript
-// 1. Generate the ID
-const awakeable = ctx.awakeable<string>();
-const id = awakeable.id
-
-// 2. Send the ID to some external system
-// ... here goes your custom code to send the ID ...
-
-// 3. Wait for the ID to returned and retrieve the payload
-const result = await awakeable.promise;
+CODE_LOAD::ts/src/develop/awakeable.ts#<start_create>-<end_create>
```

Comment:
I assume this part will fail if the specified file or tags don't exist, right?

Reply:
It does not seem to fail if the tags are not present. Maybe this is something to add to better spot problems with wrong tags.